.github/workflows/coderabbit-thread-check.yml

```
@@ -3,14 +3,143 @@ name: coderabbit-thread-check
 on:
   pull_request:
     branches: [main]
-    types: [opened, reopened, synchronize, ready_for_review]
+    types: [opened, reopened, synchronize, ready_for_review, labeled, unlabeled]
   pull_request_review:
     types: [submitted, edited, dismissed]
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)